feature_engine/parameter_checks.py

```diff
@@ -34,3 +34,76 @@ def _define_numerical_dict(dict_: Optional[dict]) -> Optional[dict]:
         raise TypeError("The parameter can only take a dictionary or None")
 
     return dict_
+
+
+def check_input_features(
+        input_features: Union[List, str], all_variables: List
+) -> Union[List, str]:
+    """
+    Checks if input_features is None; otherwise, check whether input_variables
+    is a list of features that exist within transformed dataset.
+
+    Parameters
+    ----------
+    input_features : Union[List, str]
+        The features to check whether they
+
+    all_variables: List
+        All the variables associated with the transformed dataset.
+
+    Raises
+    ------
+    ValueError
+        If input_features is not a list or all the features are not included in the transformed
+        variables.
+
+    Returns
+    -------
+    res: bool
+        Returns True if all the input_features are included in variables; otherwise,
+        returns False.
+
+    """
+    if input_features is None:
+        # return all feature names
+        feature_names = all_variables
+
+    else:
+        # Return features requested by user.
+        if not isinstance(input_features, list):
+            raise ValueError(
+                f"input_features must be a list. Got {input_features} instead."
+            )
+        if _check_input_features_in_variables(input_features, all_variables):
+            raise ValueError(
+                "Some features in input_features were not used to extract new "
+                "variables. Pass either None, or a list with the features that "
+                "were used to create date and time features."
```

Comment:
We need to replace the error message:

"Some features in input_features were not transformed by this transformer. "
"Pass either None, or a list with the features that were transformed by this transformer."